package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "debugger",
-  "displayName": "temporal-debugger-extension",
-  "description": "Temporal VSCode Debugger extension",
+  "name": "temporalio",
+  "displayName": "Temporal.io Debugger",
```

Comment:
I'm fine using this display name for now but in the future we might want to extend this functionality.

Reply:
That's true 🤔 I'll leave it as `Temporal.io`. Also looking into adding a logo file...